### ISSUE TITLE
Not working correctly with at-rule contained style definition.

### ISSUE THREAD

**Issue by subuta** [postcss/postcss-js#13]
i found postcss-js not working correctly with some style definition like below.

```javascript
// #3 not works ...
result = postcss({
    '@media (max-width: 460px)': { // first at-rule
        fontSize: 16
    },
    fontSize: 20 // second style-definition
});
```

may be the problem is definition order. i tested and other pattern and works fine :)

``` javascript
// #1 good!
result = postcss({
    fontSize: 20, // first style-definition
    '@media (max-width: 460px)': { // second at-rule
        fontSize: 16
    }
});

// #2 also good!
result = postcss({
     '@media (max-width: 460px)': { // only at-rule
         fontSize: 16
    }
});
```

[how to reproduce]
https://github.com/subuta/postcss-js-example

**Comment by ai** [comment]
Fixed 4c8a7ad

**Comment by ai** [comment]
Released in 0.1.3 version

**Comment by subuta** [comment]
@ai thanks for fix this issue! and sorry for late reply ;)
